src/Umbraco.Web.UI.Client/src/common/directives/components/umbgroupsbuilder.directive.js

```diff
@@ -484,9 +484,9 @@
 
             };
 
-            scope.openDocumentType = (documentTypeId) => {
+            scope.openContentType = (contentTypeId) => {
                 const editor = {
-                    id: documentTypeId,
+                    id: contentTypeId,
                     submit: () => {
                         const args = { node: scope.model };
                         eventsService.emit("editors.documentType.reload", args);
```

Comment:
It would be great to renamed this from `editors.documentType.reload` to `editors.contentType.reload` but I think that would be a breaking change.

Reply:
Yeah, I did see the event name, but thought that I would just let it be in case someone uses it